sale_cancel_reason/__manifest__.py

```diff
@@ -46,5 +20,5 @@
  'auto_install': False,
  'test': ['test/sale_order_cancel.yml',
```

Comment:
There are 2 main problems with YAML tests:

* They are deprecated since 2 versions, so they are going to be removed very soon (I already expected that this won't work on v10).
* The full set of things in new API is not supported via YAML.

That's why we encourage people to switch as soon as possible to Python unit tests.